crates/turborepo-lib/src/shim.rs

```diff
@@ -46,6 +46,24 @@ fn turbo_version_has_shim(version: &str) -> bool {
     version.major > 1
 }
 
+fn turbo_version_in_range(version: &str, root_path: &Path) -> bool {
+    let version = Version::parse(version).unwrap();
```

Comment:
Again, this is overly cautious, but if we somehow shipped a binary where `get_version` wasn't returning a valid semver it's helpful dump what we're failing to parse:
```suggestion
    let version = Version::parse(version).unwrap_or_else(|| panic!("Invalid semver: {}", version));
```